test/extended/cpu_partitioning/managed.go

```diff
@@ -0,0 +1,189 @@
+package cpu_partitioning
+
+import (
+	"context"
+	"time"
+
+	g "github.com/onsi/ginkgo/v2"
+	o "github.com/onsi/gomega"
+
+	appv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/labels"
+
+	exutil "github.com/openshift/origin/test/extended/util"
+	"github.com/openshift/origin/test/extended/util/image"
+)
+
+var _ = g.Describe("[sig-node] CPU Partitioned cluster workloads", func() {
```

Comment:
you need to mark that ` [apigroup:config.openshift.io]` in the name because this will fail on microshift.  There is a PR to make this easier, but you're likely to merge first.

Reply:
Sounds good, updated.